### ISSUE TITLE
JavaScript Console in Chrome not Accessible with Jaws Screen Reader

### ISSUE THREAD

**Issue by kumar140** [DJSlydeezy/google-axs-chrome#184]
When running JavaScript code in console, chrome does not allow code reading with Jaws. There's no way to navigate in the code with regular navigation keys of arrows. I am using Chrome Version 59.0.3071.115 with Jaws screen reader version 16 to run code in console.
